Main.java

```diff
@@ -1,15 +1,50 @@
-/**
- *
- * @author 
- */
-public class Main {
-
-  /**
-   * The method that is executed when you hit the run button.
-   * @param args the command line arguments
-   */
-  public static void main(String[] args) {
-    // TODO code application logic here
-    
-  }
-}
+import java.util.Scanner;
+/**
+*This program is going to ask the user for the size of pizza they want and then calculate their total with taxes labour cost and energy cost furthermore it will also display a message according to the size of the pizza
+*@author Ahmad Zeshan
+*/
+public class Main {
+
+  /**
+   * The method that is executed when you hit the run button.
+   * @param args the command line arguments
+   */
+  public static void main(String[] args) {
+		// create a Scanner for user input
+		Scanner input = new Scanner(System.in);
+		//sets the variables for the labour and energy cost
+		final double labour = 0.75;
+		final double POWER_COST = 0.99;
+		final double SIZE_COST = 0.50;
```

Comment:
You got the correct formatting for 2 out of the 3 constants here. Remember, constants should always be named using UPPERCASE_LETTERS.